frontend/apps/reader/modules/readermenu.lua

```diff
@@ -230,6 +231,51 @@ function ReaderMenu:setUpdateItemTable()
         },
     }
 
+    if not Device:isTouchDevice() then
+        -- This menu entry is a duplicate of the one found in page_turns for touch devices
+        -- but we need to add it here for non-touch devices.
+        table.insert(self.menu_items.document_settings.sub_item_table, {
+            text_func = function()
+                local text = _("Invert document-related dialogs")
+                if G_reader_settings:isTrue("invert_ui_layout") then
+                    text = text .. "   ★"
+                end
+                return text
+            end,
+            checked_func = function()
+                return self.view:shouldInvertBiDiLayoutMirroring()
+            end,
+            callback = function()
+                UIManager:broadcastEvent(Event:new("ToggleUILayoutMiroring"))
+            end,
+            hold_callback = function(touchmenu_instance)
+                local invert_ui_layout = G_reader_settings:isTrue("invert_ui_layout")
+                local MultiConfirmBox = require("ui/widget/multiconfirmbox")
+                UIManager:show(MultiConfirmBox:new{
+                    text = invert_ui_layout and _("The default (★) for newly opened books is to Invert document-related dialogs.\n\nWould you like to change it?")
+                    or _("The default (★) for newly opened books is not to Invert document-related dialogs.\n\nWould you like to change it?"),
+                    choice1_text_func = function()
+                        return invert_ui_layout and _("Don't invert") or _("Don't invert (★)")
+                    end,
+                    choice1_callback = function()
+                        G_reader_settings:makeFalse("invert_ui_layout")
+                        if touchmenu_instance then touchmenu_instance:updateItems() end
+                    end,
+                    choice2_text_func = function()
+                        return invert_ui_layout and _("Invert (★)") or _("Invert")
+                    end,
+                    choice2_callback = function()
+                        G_reader_settings:makeTrue("invert_ui_layout")
+                        if touchmenu_instance then touchmenu_instance:updateItems() end
+                    end,
+                })
+            end,
+            help_text = _([[
+When enabled the UI direction for the Table of Contents, Book Map, and Page Browser dialogs will mirror the default UI direction.
+Useful when used alongside Invert page turns.]]),
```

Comment:
I disprefer the look of that. (Disprefer is a word in common use despite what the red underline may claim. :p)
```suggestion
Useful when used alongside invert page turns.]]),
```

```suggestion
Useful when used alongside 'Invert page turns'.]]),
```